app/assets/stylesheets/splash.scss

```diff
@@ -0,0 +1,25 @@
+.splash_container {
+  // background-image: image-url('chalkboard.jpg');
+  // background-size: 100%;
+  // background-repeat: no-repeat;
+  background-color: #000000;
+  background-image: image-url('chalkboard.jpg');
+  background-size: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  position: fixed;
+  top: 0; right: 0; bottom: 0; left: 0;
+  z-index: 99999;
+}
+
+.welcome a {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.